app/models/components/course/experience_points_records_ability_component.rb

```diff
@@ -0,0 +1,49 @@
+module Course::ExperiencePointsRecordsAbilityComponent
+  include AbilityHost::Component
+
+  def define_permissions
+    # cancancan is unable to merge abilities if administrators are not excluded
```

Comment:
For abilities where we don't use hashes, "You cannot use this with multiple can definitions that match the same action and model since it is not possible to combine them. An exception will be raised when that is the case."

Cancancan will throw an error since `can :manage, :all` is defined for admins.

Will make the comment clearer.

https://github.com/ryanb/cancan/wiki/Defining-Abilities-with-Blocks#block-conditions-with-scopes